### DISCUSSION TITLE
Looking for software for music playback and management

### DISCUSSION THREAD

**Discussion by stax76** [mpvnet-player/mpv.net#439] [Q&A]
I want to give up MusicBee because it's not open source and also not portable, the UI is also a bit dated. Not sure if it's a silly idea because it works well, since a long time. I use it with a personal extension written in VB.NET.

If you know good software, please suggest. mpv/mpv.net is preferable, but alone probably not sufficient because it hasn't a database, can't edit tags etc.

I usually download channels from YouTube and delete bad tracks with the 0 button of a classic remote, rate it with 1-5 button.

**Comment by dyphire** [comment]
I have also been using musicbee because it is difficult to find a better substitute. 
But if you are looking for an open source and portable music software, [dopamine](https://github.com/digimezzo/dopamine) may be a good choice.

**Comment by stax76** [comment]
I tested some apps, including dopamine, and like that it's Electron based, but it hasn't many features.

Strawberry player is what I settled for, for tagging, but for playback I fully migrated to mpv.net! Having a bunch of scripts in place, dealing with my music needs.

**Comment by hrxn** [comment]
Something like this? https://github.com/harmonoid/harmonoid

**Reply by stax76** [reply]
Had this tested as well.

**Reply by hrxn** [reply]
You don't like it?

**Reply by stax76** [reply]
It's okay, uses Dart, Flutter and mpv, it looks good, uses a modern UI framework like dopamine.

I choose strawberry because it's feature rich, maybe closest to MusicBee feature wise, others might work too, I don't have a lot of requirements, just adding some tags, playback does mpv.net now.

With MusicBee I was using ratings, but that might only make sense with a grid view, tabs, filters etc., which mpv.net don't have, so I decided to give up ratings and use 2 folders, `Favorites` and `More`, ratings aren't supported well in most music apps, some support it, but not in a compatible way.
